PanelViewController/ContainerViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
```

Comment:
Why was this project started?  Was it in response to an issue and if so which one?

Reply:
Removed support for non-animated state change